src/main/java/pl/jablonskanycz/bakery/BakeryApplication.java

```diff
@@ -1,29 +1,129 @@
 package pl.jablonskanycz.bakery;
 
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import pl.jablonskanycz.bakery.products.bread.Bread;
-import pl.jablonskanycz.bakery.products.bread.WholegrainBreadFactory;
-import pl.jablonskanycz.bakery.products.bun.Bun;
-import pl.jablonskanycz.bakery.products.bun.RhubarbBunFactory;
-import pl.jablonskanycz.bakery.products.bun.SesameBunFactory;
-import pl.jablonskanycz.bakery.products.bun.AsparagusBun;
-
-import java.time.Instant;
+
+import pl.jablonskanycz.bakery.clients.Address;
+import pl.jablonskanycz.bakery.clients.Client;
+import pl.jablonskanycz.bakery.clients.ListBasedAddressRepository;
+import pl.jablonskanycz.bakery.clients.ListBasedClientRepository;
+import pl.jablonskanycz.bakery.products.ListBasedProductRepository;
+import pl.jablonskanycz.bakery.products.Product;
+import pl.jablonskanycz.bakery.products.ProductType;
+import pl.jablonskanycz.bakery.products.bread.*;
+import pl.jablonskanycz.bakery.products.bun.*;
+
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
 
 @SpringBootApplication
 public class BakeryApplication {
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
 
 //        SpringApplication.run(BakeryApplication.class, args);
 
+//Data import
+        ListBasedAddressRepository addressRepository = new ListBasedAddressRepository();
+        ListBasedClientRepository clientRepository = new ListBasedClientRepository();
+        ListBasedProductRepository productRepository = new ListBasedProductRepository();
+        ListBasedEmployeeRepository employeeRepository = new ListBasedEmployeeRepository();
+
+        Path addressPath = Path.of("src", "main", "resources", "ADDRESS.csv");
+        Path clientPath = Path.of("src", "main", "resources", "CLIENT.csv");
+        Path employeePath = Path.of("src", "main", "resources", "EMPLOYEE.csv");
+        Path productPath = Path.of("src", "main", "resources", "PRODUCT.csv");
+
+        List<Address> addresses = Files.lines(addressPath)
+                .skip(1)
+                .map(line -> {
+                    String[] address = line.split(",");
+                    return new Address(Integer.parseInt(address[0]), Double.parseDouble(address[1]), Double.parseDouble(address[2]));
+                })
+                .collect(Collectors.toList());
+
+        for (Address address : addresses) {
```

Comment:
to answer your question from email - we could just do `addAddress` inside the `map` above (however let's first try to extract methods to a new, separate repository)